dbt_subprojects/dex/macros/models/_project/uniswap_compatible_trades.sql

```diff
@@ -118,4 +118,74 @@ SELECT
     , dexs.evt_index
 FROM
     dexs
+{% endmacro %}
+
+{% macro uniswap_compatible_v4_trades(
+    blockchain = null
+    , project = 'uniswap'
+    , version = '4'
+    , Pair_evt_Swap = null
+    , Factory_evt_PoolCreated = null
```

Comment:
```suggestion
    , PoolManager_evt_Swap = null
    , PoolManager_evt_Initialize = null
```

Reply:
to hosuke's point, since naming standards changed across versions, makes sense to keep param names clean